### PR TITLE
Fixes test stability by rounding forecast values

### PR DESCRIPTION
**Pull request by merschformann**
# Description
Stabilizes test of forecasting app by rounding the forecast values.

## Changes
- Added instructions for updating golden files in README.
- Updated forecast app golden file test expectation.
- Introduced output processing configuration for rounding forecast values to 7 decimal places in `main_test.go`.
- Updated Nextmv SDK version to `v1.6.2` in `go.mod` (new rounding and list support features).

### CODE REVIEW

File: .nextmv/golden/demand-forecasting-ortools/main_test.go [hunk 1]
```diff
@@ -31,6 +31,14 @@ func TestGolden(t *testing.T) {
 					Replacement: golden.StableFloat,
 				},
 			},
+			OutputProcessConfig: golden.OutputProcessConfig{
+				RoundingConfig: []golden.RoundingConfig{
+					{
+						Key:       ".solutions[].forecast",
+						Precision: 7,
```
Comment (sebastian-quintero) [inline .nextmv/golden/demand-forecasting-ortools/main_test.go:38]: LOL

File: .nextmv/golden/go.mod [hunk 2]
```diff
@@ -2,7 +2,7 @@ module apps-golden-file-tests
 
 go 1.22
 
-require github.com/nextmv-io/sdk v1.5.0
+require github.com/nextmv-io/sdk v1.6.2-0.20240607121831-c1317be402f1
```
Comment (sebastian-quintero) [inline .nextmv/golden/go.mod]: We should probably release `sdk` sometime soon.

**Review by sebastian-quintero — APPROVED** [review]

File: .nextmv/golden/demand-forecasting-ortools/main_test.go (hunk 1 above)
Comment (merschformann) [reply .nextmv/golden/demand-forecasting-ortools/main_test.go:38]: Yeah, updating the tests annoyed me. :sweat_smile:

File: .nextmv/golden/go.mod (hunk 2 above)
Comment (merschformann) [reply .nextmv/golden/go.mod]: Done

**Pull request merged**